guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc

```diff
@@ -1,7 +1,10 @@
 [id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]
 = Exporting a Content View Version in a Syncable Format
 
-You can export a version of a Content View to a syncable format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
+You can export a version of a Content View to a syncable format that you can use to create your custom CDN. Once exported, do either of the following:
+
+* Synchronize the content from the custom CDN over HTTP/HTTPS.
+* Import the content via `hammer content-import` which is available if both the Export and Import servers are {ProductVersion} and above.
```

Comment:
```suggestion
* Import the content using `hammer content-import`.
Note that this requires both the Export and Import servers are {ProductVersion} and above.
```

I am unsure about the second sentence because the docs are specifically built for {ProductVersion} anyway. Maybe reword so that users ensure both the export and import {ProjectServer} are on the same/target version?